config/pod.yaml.tim

```diff
@@ -21,22 +21,16 @@ spec:
     command:
     - "/opt/sd/launcher_entrypoint.sh"
     args:
-    # Run the launcher and logservice in the background
-    # Wait for both background jobs to complete
+    # Run the wrapper script
     - |
-      /opt/sd/launch --api-uri {{api_uri}} --store-uri {{store_uri}} --emitter /opt/sd/emitter --build-timeout {{build_timeout}} {{build_id}} &
-      /opt/sd/logservice --emitter /opt/sd/emitter --api-uri {{api_uri}} --store-uri {{store_uri}} --build {{build_id}} &
-      wait $(jobs -p)
+      /opt/sd/run.sh "{{token}}" {{api_uri}} {{store_uri}} {{build_timeout}} {{build_id}}
```

Comment:
can you double quote these shell variables as well ?

Reply:
Fixed it.